service-orchestration/README.md

```diff
@@ -0,0 +1,622 @@
+
+1. Receive a request with a JSON payload in the following form.
+
+```json
+{
+    "patient": {
+        "name": "John Doe",
+        "dob": "1940-03-19",
+        "ssn": "234-23-525",
+        "address": "California",
+        "phone": "8770586755",
+        "email": "johndoe@gmail.com",
+        "cardNo": "7844481124110331"
+    },
+    "doctor": "thomas collins",
+    "hospital_id": "grandoaks",
+    "hospital": "grand oak community hospital",
+    "appointment_date": "2023-10-02"
+}
+```
+
+2. Extract necessary details from the request (e.g., hospital, patient, doctor, etc.) and make a call to the hospital backend service to request an appointment. A response similar to the following will be returned from the hospital backend service on success.
+
+```json
+{
+    "appointmentNumber": 1,
+    "doctor": {
+        "name": "thomas collins",
+        "hospital": "grand oak community hospital",
+        "category": "surgery",
+        "availability": "9.00 a.m - 11.00 a.m",
+        "fee": 7000
+    },
+    "patient": {
+        "name": "John Doe",
+        "dob": "1940-03-19",
+        "ssn": "234-23-525",
+        "address": "California",
+        "phone": "8770586755",
+        "email": "johndoe@gmail.com"
+    },
+    "hospital": "grand oak community hospital",
+    "confirmed": false,
+    "appointmentDate": "2023-10-02"
+}
+```
+
+3. Use the hospital ID and the appointment number and call the hospital backend service to retrieve the fee for the appointment. The response will be similar to the following.
+
+```json
+{
+    "patientName": "John Doe",
+    "doctorName": "thomas collins",
+    "actualFee": "7000"
+}
+```
+
+4. Finally, call the payment backend service to make the payment and retrieve the reservation status. Log some information and respond to the client with the reservation status as the payload.
+
+```json
+{
+    "appointmentNo": 2,
+    "doctorName": "thomas collins",
+    "patient": "John Doe",
+    "actualFee": 7000,
+    "discount": 20,
+    "discounted": 5600.0,
+    "paymentID": "f130e2ed-a34e-4434-9b40-6a0a8054ee6b",
+    "status": "settled"
+}
+```
+
+### Concepts covered
+
+- REST API
+- HTTP client
+
+## Develop the application
+
+### Step 1: Set up the workspace
+
+Install [Ballerina Swan Lake](https://ballerina.io/downloads/) and the [Ballerina Swan Lake VS Code extension](https://marketplace.visualstudio.com/items?itemName=wso2.ballerina) on VS Code.
+
+### Step 2: Develop the service
+
+Follow the instructions given in this section to develop the service.
+
+1. Create a new Ballerina project using the `bal` command and open it in VS Code.
+
+```bash
+$ bal new service-orchestration
+```
+
+2. Introduce the source code in files with the `.bal` extension (e.g., the `main.bal` file). 
+
+Import the 
+- `ballerina/http` module to develop the REST API and define the clients that can be used to send requests to the backend services
+- `ballerina/log` module to log debug/error information for each client request
```

Comment:
```suggestion
- `ballerina/log` module to log debug, error, or info level information for each client request
```

I think better to have this even though the tutorial doesn't contain the `info` level logging

Reply:
Not required, right? We are explaining why we need to import those modules for this tutorial.